docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx

```diff
@@ -87,52 +87,47 @@ The Frame Processor Plugin will be exposed to JS through the `VisionCameraProxy`
 
 ![Xcode "Create Bridging Header" alert](https://docs-assets.developer.apple.com/published/7ebca7212c/2a065d1a-7e53-4907-a889-b7fa4f2206c9.png)
 
-3. Inside the newly created Bridging Header, add the following code:
-
-```objc
-#import <VisionCamera/FrameProcessorPlugin.h>
-#import <VisionCamera/Frame.h>
-```
-
-4. In the Swift file, add the following code:
+3. In the Swift file, add the following code:
 
 ```swift
+import VisionCamera
+
 @objc(FaceDetectorFrameProcessorPlugin)
 public class FaceDetectorFrameProcessorPlugin: FrameProcessorPlugin {
-  public override func callback(_ frame: Frame!, withArguments arguments: [String:Any]) -> Any {
+  public override func callback(_ frame: Frame, withArguments arguments: [AnyHashable : Any]?) -> Any {
```

Comment:
Is it really `AnyHashable`? I thought it'd always be string keys, but let's keep it at AnyHashable - good point on the nullable part

Reply:
The Swift module (`import VisionCamera`) translates ObjC generic `NSDictionary` into `[AnyHashable : Any]`

<img width="931" alt="Screenshot 2023-09-06 at 16 11 30" src="https://github.com/mrousavy/react-native-vision-camera/assets/25980166/2d8d30d4-50ae-4680-bdf1-b243061a648a">